processing/data_collection/gazette/spiders/instar_base.py

```diff
@@ -0,0 +1,34 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BaseInstarSpider(BaseGazetteSpider):
+    def parse(self, response):
+        page_url = response.urljoin("{page}/0/0/0/0/")
+        last_page = int(
+            response.xpath("//select[@id='select']/option[last()]/text()").get()
+        )
+        for page in range(2, 1 + last_page):
+            yield Request(page_url.format(page=page), callback=self.parse_editions_page)
```

Comment:
Cities like Camaquã (MG) don't have pagination (yet) and break at this selection. We could solve it by doing something like:
```suggestion
        last_page = response.xpath("//select[@id='select']/option[last()]/text()").get()
      	if last_page is not None:
        	for page in range(2, 1 + int(last_page)):
            	yield Request(page_url.format(page=page), callback=self.parse_editions_page)
```